templates/node-class.mustache

```diff
@@ -18,6 +18,8 @@
         if(this.domain.length === 0) {
             throw new Error('Domain parameter must be specified as a string.');
         }
+        {{#isES6}}let{{/isES6}}{{^isES6}}var{{/isES6}} headers = (typeof options === 'object') ? options.headers : {};
```

Comment:
You can use `const` instead of `let` since you never change this variable.

Reply:
You are technically right, but I decided to follow the same coding style used to define the `domain` attribute (line 16) in order introduce as fewer changes as possible and to establish a pattern that could be easily refactored in the future.